library/std/src/sys/dbg.rs

```diff
@@ -0,0 +1,129 @@
+//! Debugging aids.
+//!
+//! On Windows, the approach is based on the Structured Exception Handling:
+//! https://learn.microsoft.com/en-us/windows/win32/debug/structured-exception-handling
+//! and is conceptually equivalent to
+//!
+//! ```c
+//! __try {
+//!     __debugbreak();
+//! } __except (1) {
+//!     /* Nothing */
+//! }
+//! ```
+//!
+//! The implementation is available for the `x86_64` and the `aarch64`
+//! targets under Windows.
+
+#[cfg(target_os = "windows")]
+mod windows {
+    #[cfg(not(target_arch = "x86"))]
+    extern "C" {
+        /// Breakpoint that is passed to the debugger as the first chance exception
+        /// if the debugger is attached, and is skipped over otherwise.
+        pub(super) fn __dbg_breakpoint();
+    }
+
+    #[cfg(target_arch = "x86_64")]
+    core::arch::global_asm!(
+        r#"
+        .pushsection    .text
+
+        .globl          __dbg_breakpoint_flt
```

Comment:
It is not the 1980s and we do not need to conserve singular bytes in source code anymore, much less in a one-off case like this one.
```suggestion
        .global          __dbg_breakpoint_flt
```